filament/backend/src/opengl/OpenGLProgram.h

```diff
@@ -108,10 +121,12 @@ class OpenGLProgram : public HwProgram {
         mutable uint16_t age = std::numeric_limits<uint16_t>::max();
     };
     UniformsRecord const* mUniformsRecords = nullptr;
+
+    utils::FixedCapacityVector<PushConstantBundle> mVertexPushConstants;
 };
 
 // if OpenGLProgram is larger tha 64 bytes, it'll fall in a larger Handle bucket.
-static_assert(sizeof(OpenGLProgram) <= 64); // currently 48 bytes
+static_assert(sizeof(OpenGLProgram) <= 64); // currently 54 bytes
```

Comment:
I think you can keep the size below 64 bytes even if you handle both stages, by storing the info for both stages in the same vector, you just need a uint8_t to store the offset (we have 3 bytes padding available) and change the API a bit to return a range (you can use utils::Slice) when getting the constants.

Reply:
done, but didn't use `utils::Slice` and just did simple arithmetic